### ISSUE TITLE
Left handed friendly console (ctrl+insert, shift+insert)

### ISSUE THREAD

**Issue by leonardommarques** [rstudio/rstudio#3088]
### System details

    RStudio Edition : 
    RStudio Version : 1.1.453
    OS Version      : Windows7
    R Version       : 3.5.0

### Steps to reproduce the problem

 - select/highlight a text in the console
 - press ctrl+insert

### Describe the problem in detail

ctrl+insert dos not work as an alternative to ctrl+c to copy content from console

### Describe the behavior you expected

I expected the hilighted content in the console to be copied into the clipboard

**Comment by ronblum** [comment]
@leonardommarques Thank you for reporting this issue. Are you using the standalone RStudio Desktop or connecting to RStudio Server through a web browser?

**Comment by leonardommarques** [comment]
Hi. I am using RStudio Desktop

**Comment by leonardommarques** [comment]
Do you need any other info on that?

**Comment by ronblum** [comment]
@leonardommarques No, the information you've provided is good. We have many issues to go through, unfortunately. I'm marking it as something for us to try to reproduce since you've provided what we need to know.

In the meantime, [the current RStudio release, 1.2.5019,](https://www.rstudio.com/products/rstudio/download/) was updated recently. In addition,
[a preview of the upcoming update is available](https://www.rstudio.com/products/rstudio/download/preview/), but note that preview builds are intended for testing purposes only.

**Comment by ronblum** [comment]
I can confirm this is an issue with RStudio Desktop 1,1.453 and 1.3.895 on Windows 10.

**Comment by gtritchie** [comment]
Marking as an accessibility issue due to keyboard-shortcut angle.

**Comment by leonardommarques** [comment]
So?

**Comment by ronblum** [comment]
Unstaling due to the accessibility concern.

**Comment by leonardommarques** [comment]
so?
